src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py

```diff
@@ -157,3 +160,15 @@ def _parameters(registry_name,
     if customized_api_version:
         parameters['registryApiVersion'] = {'value': customized_api_version}
     return parameters
+
+def random_storage_account_name(registry_name, storage_client=None):
+    if storage_client is None:
+        storage_client = get_storage_service_client().storage_accounts
+
+    random_suffix = ''.join(choice(digits) for i in range(3))
```

Comment:
the randomness here doesn't seem to be sufficient to me. chance of conflict is fairly high comparing to the original uuid approach. if the what actually needed is 'uniqueness' rather than 'randomness' what about using a simple timestamp here? 
``` Python
datetime.utcnow().strftime('%Y%m%d%H%M%S')
```